src/couch/src/couch_secondary_sup.erl

```diff
@@ -22,12 +22,12 @@ init([]) ->
         {couch_plugin_event, {gen_event, start_link, [{local, couch_plugin}]}, permanent,
             brutal_kill, worker, dynamic}
     ],
-    Daemons = [
-        {index_server, {couch_index_server, start_link, []}},
-        {query_servers, {couch_proc_manager, start_link, []}},
-        {vhosts, {couch_httpd_vhost, start_link, []}},
-        {uuids, {couch_uuids, start, []}}
-    ],
+    Daemons =
+        [
```

Comment:
erlfmt did this... for some reason.

Reply:
I don't mind how it's formatted, I'm just saying that erlfmt reformatted it, so you don't go looking for deeper meaning to this part of the diff. The only relevant change is the removal of the single `index_server` and the addition of the `index_servers()` list.

Reply:
Ah ok, I see. Sometimes erlfmt formats based on a larger context. So, for example, a list with two elements might not get re-formatted but once a third one is added, it will reformat the whole list.